target/linux/apm821xx/image/Makefile

```diff
@@ -122,7 +122,8 @@ endef
 ifeq ($(SUBTARGET),nand)
 
 define Device/meraki_mr24
-  DEVICE_TITLE := Cisco Meraki MR24
+  DEVICE_VENDOR := Cisco
```

Comment:
Problem here is that Meraki was different company and it produced the MR24 before they got bought by Cisco. So there are some with the original Meraki and some with just a liitle cisco bridge logo. Modelling this would require something like https://github.com/openwrt/openwrt/pull/2250 first.

As for the "Cisco". From what I know, Cisco decided to keep the "MerakI" brand around for these and future devices: https://en.wikipedia.org/wiki/Cisco_Meraki

Reply:
We can also make use of `DEVICE_ALT*` variables here already. As is done here e.g. https://github.com/openwrt/openwrt/pull/2229/files#diff-b436f01932a18876c27800ba183d95f6R142
```
DEVICE_VENDOR := Cisco
DEVICE_MODEL := MR24
DEVICE_ATL0_VENDOR := Meraki
DEVICE_ATL0_MODEL := MR24
```
Let me know if that is ok.

Reply:
@chunkeey "Cisco Meraki" is a company on it's own, so I would argue that can set that as DEVICE_TITLE.